### PR TITLE
Document repo analytics

### PR DESCRIPTION
**Pull request by cgarciae**
# What does this PR do?
Documents how to run the repo analytics and does some cleanup.

### CODE REVIEW

File: .github/analytics/get_repo_metrics.py [hunk 1]
```diff
@@ -407,4 +408,4 @@ def main(
   plt.show()
 
 if __name__ == '__main__':
-  main()
+  typer.run(main)
```
Comment (gnecula) [inline .github/analytics/get_repo_metrics.py:411]: I believe that we use "from absl import app" and "app.main" in other places in the project to run the main function. Perhaps then we do not need the `typer` dependency.

Comment (cgarciae) [reply .github/analytics/get_repo_metrics.py:411]: Let me try changing it to `absl` for consistency.

Comment (cgarciae) [reply .github/analytics/get_repo_metrics.py]: Ok, update to use `absl`.

**Comment by cgarciae** [comment]
Thanks @gnecula for reviewing it. Updated the script to use `absl`.

**Review by gnecula — APPROVED** [review]

**Pull request merged**